tests/Testcontainers.Spanner.Tests/.editorconfig

```diff
@@ -0,0 +1,241 @@
+root = true
```

Comment:
Here applies the same as for the `.editorconfig` in `src`.

Reply:
will adapt this